db/upgrade-X.X.X-X.Y.Z.sql

```diff
@@ -49,3 +49,16 @@ call ValidateVersion;
 
 
 INSERT INTO pf_version (id, version) VALUES (@VERSION_INT, CONCAT_WS('.', @MAJOR_VERSION, @MINOR_VERSION, @SUBMINOR_VERSION));
+
+
+
+
+
+
+---
+--- https://github.com/inverse-inc/packetfence/pull/2785
+--- 
+ALTER TABLE `radius_audit_log` ADD KEY (`auth_status`, `created_at`);
```

Comment:
You'll need to also modify `pf-schema-X.Y.Z.sql`. The `created_at` key seems already defined.

Reply:
The single KEY (created_at) isn't useful when using a compound WHERE

        WHERE `auth_status`='Accept'
            AND `created_at` BETWEEN ? AND ?

A second compound KEY (`auth_status`,`created_at`) is useful for the compounded WHERE above.

Reply:
Schema added in https://github.com/inverse-inc/packetfence/pull/2785/commits/e1e09a6cde3ecceea23e7da6eeb30590e011136c